packages/nodes-base/nodes/Aws/Cognito/descriptions/user/User.resource.ts

```diff
@@ -0,0 +1,229 @@
+					},
+				},
+			},
+			{
+				name: 'Delete',
+				value: 'delete',
+				description: 'Delete a user',
+				action: 'Delete user',
+				routing: {
+					send: {
+						preSend: [preSendUserFields],
+					},
+					request: {
+						method: 'POST',
+						headers: {
+							'X-Amz-Target': 'AWSCognitoIdentityProviderService.AdminDeleteUser',
+						},
+						ignoreHttpStatusErrors: true,
+					},
+					output: {
+						postReceive: [
+							handleError,
+							{
+								type: 'set',
+								properties: {
+									value: '={{ { "deleted": true } }}',
+								},
+							},
+						],
+					},
+				},
+			},
+			{
+				name: 'Get',
+				value: 'get',
+				description: 'Retrieve information of an existing user',
+				action: 'Get user',
+				routing: {
+					send: {
+						preSend: [preSendUserFields],
+					},
+					request: {
+						method: 'POST',
+						headers: {
+							'X-Amz-Target': 'AWSCognitoIdentityProviderService.AdminGetUser',
+						},
+						ignoreHttpStatusErrors: true,
+					},
+					output: {
+						postReceive: [handleError, simplifyUser],
+					},
+				},
+			},
+			{
+				name: 'Get Many',
+				value: 'getAll',
+				description: 'Retrieve a list of users',
+				routing: {
+					request: {
+						method: 'POST',
+						headers: {
+							'X-Amz-Target': 'AWSCognitoIdentityProviderService.ListUsers',
+						},
+						ignoreHttpStatusErrors: true,
+					},
+					output: {
+						postReceive: [
+							handleError,
+							simplifyUser,
+							{
+								type: 'rootProperty',
+								properties: {
+									property: 'Users',
+								},
+							},
+						],
+					},
+				},
+				action: 'Get many users',
+			},
+			{
+				name: 'Remove From Group',
+				value: 'removeFromGroup',
+				description: 'Remove a user from a group',
+				action: 'Remove user from group',
+				routing: {
+					send: {
+						preSend: [preSendUserFields],
+					},
+					request: {
+						method: 'POST',
+						headers: {
+							'X-Amz-Target': 'AWSCognitoIdentityProviderService.AdminRemoveUserFromGroup',
+						},
+						ignoreHttpStatusErrors: true,
+					},
+					output: {
+						postReceive: [
+							handleError,
+							{
+								type: 'set',
+								properties: {
+									value: '={{ { "removedFromGroup": true } }}',
+								},
+							},
+						],
+					},
+				},
+			},
+			{
+				name: 'Update',
+				value: 'update',
+				description: 'Update an existing user',
+				action: 'Update user',
+				routing: {
+					send: {
+						preSend: [preSendUserFields],
+					},
+					request: {
+						method: 'POST',
+						headers: {
+							'X-Amz-Target': 'AWSCognitoIdentityProviderService.AdminUpdateUserAttributes',
+						},
+						ignoreHttpStatusErrors: true,
+					},
+					output: {
+						postReceive: [
+							handleError,
+							{
+								type: 'set',
+								properties: {
+									value: '={{ { "updated": true } }}',
+								},
+							},
+						],
+					},
+				},
+			},
+		],
+	},
+
+	...create.description,
+	...del.description,
```

Comment:
please use the same naming convention to keep consistency 
```suggestion
	...delete.description,
```